### ISSUE TITLE
Custom CSS no longer affects the feed view with 2.7

### ISSUE THREAD

**Issue by bonesbro** [brief-rss/brief#539]
After the newest update the Custom CSS option no longer affects the feed view.

Repro steps:
1. Open Brief
2. Options > Options
3. Enter some custom CSS, like body { background-color:#b0b0a7; }
4. Close the options pane

Expected: The CSS applies and the background color of the feed pane turns darker
Actual: No impact

Looking in the debug panel, the main page has this CSS reference:

`<link xmlns="http://www.w3.org/1999/xhtml" id="custom-css" rel="stylesheet" type="text/css" href="blob:moz-extension://d2ab27ab-fc90-45d0-ad90-2c7a2a5afed2/59698a91-1351-4c4b-85dd-05be64b8ddeb" />`

The feed view iframe has a custom-css element but it's always empty:

`<link id="custom-css" rel="stylesheet" type="text/css">`

Using the F12 tools I can manually copy the parent frame's stylesheet reference into the child frame and everything appears to work properly.

Example CSS that I'm using:
```
body { background-color:#b0b0a7; }
.full-container { background-color:#E8E8DF; flex: 0 1 1000px;}
.content { font-family:Calibri; font-size:11.5pt; }
```

Additional bug: the "Learn about customizing with CSS" link in the options page 404s.

**Comment by tanriol** [comment]
Please check whether your Brief version is 2.7.0 or 2.7.1 - there was a bug in 2.7.0 in custom CSS handling that's been fixed in 2.7.1.

**Comment by bonesbro** [comment]
Thank you!  I had 2.7.0, and I've upgraded to 2.7.1 and all is well again.
